### PR TITLE
#92 and some more

### PR DESCRIPTION
**Pull request by xtrnkal**

### CODE REVIEW

File: web/src/main/webapp/WEB-INF/jsp/home.jsp [hunk 1]
```diff
@@ -12,7 +12,7 @@
             <div class="current-shows-panel row up-to-large-5">
                 <c:forEach items="${shows}" var="show">
                 <div class="img-container column">
-                    <my:a href="show/${show.id}">
```
Comment (rutor) [inline web/src/main/webapp/WEB-INF/jsp/home.jsp:15]: show is correct path

**Review by rutor — CHANGES_REQUESTED** [review]
/show_detail/* is not mapped in controller

File: web/src/main/webapp/WEB-INF/jsp/home.jsp (hunk 1 above)
Comment (xtrnkal) [reply web/src/main/webapp/WEB-INF/jsp/home.jsp:15]: show_detail is correct path

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java [hunk 2]
```diff
@@ -47,11 +67,49 @@ public String view(@PathVariable long id, Model model) {
         return "show_detail";
     }
     
-    @RequestMapping(value = "/bookings_tickets", method = RequestMethod.GET)
-    public String tickets(@RequestParam(required = true) Long userId,Model model) {
-        model.addAttribute("tickets", ticketFacade.getByUser(userId));
+    @RequestMapping(value = "/show_detail/{id}", method = RequestMethod.POST)
```
Comment (rutor) [inline web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java]: Proc `/show_detail/id` vyzaduje CreateBookingDTO?

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java [hunk 3]
```diff
@@ -47,11 +67,49 @@ public String view(@PathVariable long id, Model model) {
...
+                         Model model, RedirectAttributes redirectAttributes, UriComponentsBuilder uriBuilder) {
+        log.debug("create(dto={})", dto);
+        if (bindingResult.hasErrors()) {
+            ViewErrorUtils.handleErrorInBinding(bindingResult, model);
+            return "show_detail/"+id;
+        }
+        
+        UserDTO userDTO = userFacade.getUserById(1l);
+        dto.setUser(userDTO);
+        Long idBook = bookingFacade.create(dto);
```
Comment (rutor) [inline web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java]: Proc `/show_detail` neco vytvari?

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java [hunk 4]
```diff
@@ -47,11 +67,49 @@ public String view(@PathVariable long id, Model model) {
...
+    
+    @RequestMapping(value = "/buy_ticket/{id}", method = RequestMethod.GET)
+    public String buy(@PathVariable long id, Model model) {
+        model.addAttribute("performance", performanceFacade.getPerformanceById(id));
+        
+        
+        return "buy_ticket";
+    }
+    
+    @RequestMapping(value = "/bookings_tickets/{id}", method = RequestMethod.GET)
```
Comment (rutor) [inline web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java]: ID v adrese je bezpecnostni riziko. V session je ulozeny autorizovany uzivatel, tak id vytahni z toho objektu

**Review by rutor — DISMISSED** [review]

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java (hunk 2 above)
Comment (xtrnkal) [reply web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java]: old code, removed

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java (hunk 3 above)
Comment (xtrnkal) [reply web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java]: old code, removed

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java (hunk 4 above)
Comment (xtrnkal) [reply web/src/main/java/cz/muni/fi/pa165/mvc/controllers/HomeController.java]: old code, removed

**Review by rutor — APPROVED** [review]
OK

**Pull request merged**